package.json

```diff
@@ -53,10 +53,12 @@
     "@antfu/utils": "^0.7.2",
     "@rollup/pluginutils": "^5.0.2",
     "debug": "^4.3.4",
+    "echarts": "^5.4.2",
     "fs-extra": "^11.1.1",
     "kolorist": "^1.7.0",
     "sirv": "^2.0.2",
-    "ufo": "^1.1.1"
+    "ufo": "^1.1.1",
+    "vue-echarts": "^6.5.4"
```

Comment:
<img width="1144" alt="image" src="https://user-images.githubusercontent.com/11247099/231593389-583820df-0883-44b2-b73c-0daa22b81fc0.png">

👀😆

Reply:
Ah I forgot to move it to devDependencies 😭